### PR TITLE
Add compat data for BasicCardRequest

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/BasicCardRequest

### CODE REVIEW

File: api/BasicCardRequest.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "56",
+            "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          },
+          "firefox_android": {
+            "version_added": "56",
+            "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+            },
+            "firefox_android": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+            },
+            "firefox_android": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: Use the "flags" property instead of a "notes" entry.

File: api/BasicCardRequest.json [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+          "firefox": {
+            "version_added": "56",
+            "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
+          },
+          "firefox_android": {
+            "version_added": "56",
+            "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
+          },
+          "ie": {
+            "version_added": null
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: This is definitively not in IE: false

File: api/BasicCardRequest.json [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+            "firefox": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
+            },
+            "firefox_android": {
+              "version_added": "56",
+              "notes": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
+            },
+            "ie": {
+              "version_added": null
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: This is definitively not in IE: false

File: api/BasicCardRequest.json [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
```
Comment (teoli2003) [inline api/BasicCardRequest.json]: This is definitively not in IE: false

File: api/BasicCardRequest.json (hunk 1 above)
Comment (bunnybooboo) [reply api/BasicCardRequest.json]: Like this?
```
"firefox": {
            "version_added": "56",
            "flags": [
              {
                "type": "preference",
                "name": "Hidden behind the <code>dom.payments.request.enabled</code> pref."
              }
            ]
          },
```

Comment (teoli2003) [reply api/BasicCardRequest.json]: Almost, for "name" just "dom.payments.request.enabled"; followe by "value_to_set": "true"

See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags

**Pull request merged**